frontend/pages/organizations/[id]/about.vue

```diff
@@ -47,30 +56,42 @@
           <MediaImageCarousel :class="{ 'lg:hidden': textExpanded }" />
         </div>
       </div>
-      <CardOrgApplicationVote
-        v-if="organization.status === 'pending'"
-        @up-vote="upVotes++"
-        @down-vote="downVotes++"
-        title="Votes in favor"
-        :organizations="organizationsInFavor"
-        :up-votes="upVotes"
-        :down-votes="downVotes"
+      <CardGetInvolved
+        v-if="organization.status === 'approved'"
+        :organization="organization"
       />
-      <CardGetInvolved :organization="organization" />
       <CardConnect
         :social-links="organization.socialLinks"
         :userIsAdmin="true"
       />
-      <!-- <CardDonate
+      <CardDonate
+        v-if="organization.status === 'approved'"
         :userIsAdmin="true"
         :donationPrompt="organization.donationPrompt"
-      /> -->
+      />
+
+      <h3 class="text-left responsive-h3 font-display">
+        {{ $t("_global.discussion") }}
+      </h3>
+      <hr />
+
+      <CardDiscussionText
```

Comment:
Rather than doing the for loop within the component, we'd instead want the loop to happen on the given page. We'll make a `Discussion` component at some point that has all of the components here that then does the for loop and has an input that can be disabled based on if the user is alloed to participate or not. Maybe this is something you'd want to work on next, @UnknownSean8? I can make an issue for this!